src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
@slfritchie This would just be a scan of the header of each log field to check if the checksum field for the entire file is set to enabled or not. Probably have to do that in the future to make sure all the logfiles are of the correct version anyway.

@no2chem Yes, it's important. I mean in your single test you saw a 34% overhead, so that is pretty significant. I also don't think it adds to the complexity of the protocol - (1) you're going to need flexibility in the mechanism in the future (2) currently it's a single field in the header that determines which checksum mechanism to use (currently none or CRC32 - in the future, we could add support for Intel's SHA1 extensions, for example). (3) We want to support end-to-end checksums in the end, but this PR doesn't add it.